ci-operator/step-registry/workers-rhel/aws-provision/workers-rhel-aws-provision-commands.sh

```diff
@@ -128,11 +128,16 @@ cat > create_machineset.yaml <<-'EOF'
     machineset: "{{ machineset_obj | combine(dict_edit, recursive=True) }}"
   vars:
     ssh_key_name: "{{ lookup('env', 'SSH_KEY_NAME') }}"
+    machineset_replicas: "{{ lookup('env', 'RHEL_WORKER_COUNT_PER_ZONE') }}"
```

Comment:
I did not find out the ZONE setting in machineset yaml, you can help point me there.

Reply:
@jianlinliu In this AWS RHEL machine provisioning process, it's updating the existing RHCOS worker machineset file one by one and using them to create RHEL machines, because the each RHCOS worker `machineset` itself was created per Zones, so I choose this name. 

For example, if there're three RHCOS machinesets in the beginning, 
```
[root@preserve-gpei-worker k_files]#   oc get machinesets -n openshift-machine-api
NAME                               DESIRED   CURRENT   READY   AVAILABLE   AGE
gpei-415-wc42b-worker-us-east-2a   1         1         1       1           123m
gpei-415-wc42b-worker-us-east-2b   1         1         1       1           123m
gpei-415-wc42b-worker-us-east-2c   1         1         1       1           123m
```

In the previous code, the RHEL worker number would be the same "3" as RHCOS machines, we have no way to customize the RHEL worker number but some team member is requesting this. So if we use this `RHEL_WORKER_COUNT_PER_ZONE` parameter to specify the `machineset_replicas`, such as `RHEL_WORKER_COUNT_PER_ZONE=3`, we'll get 3*3=9 RHEL workers.

Reply:
Got it, thanks for the detailed explanation.